cranelift/filetests/filetests/isa/x64/simd-bitwise-compile.clif

```diff
@@ -496,28 +495,21 @@ block0:
 ;   pushq %rbp
 ;   movq %rsp, %rbp
 ; block1: ; offset 0x4
-;   movdqu 0x34(%rip), %xmm0
+;   movdqu 0x24(%rip), %xmm0
 ;   psrlw $1, %xmm0
-;   movdqu 0x17(%rip), %xmm3
-;   pand %xmm3, %xmm0
+;   pand 7(%rip), %xmm0
```

Comment:
Interesting that this isn't `0x7` but maybe the printer doesn't care because it doesn't really matter?

Reply:
Yeah I think this is capstone so my guess is that a less-than-10 heuristic or something like that is kicking in